en/faq.md

```diff
@@ -6,23 +6,23 @@ sidebarDepth: 2
 
 [[toc]]
 
-## 站点使用相关
+## Station usage related
 
-### 收不到验证邮件？
+### Can't receive verify email？
 
-请先检查你的邮箱的垃圾箱。如果垃圾箱中也没有验证邮件，尝试再发送一次验证邮件。
+Please check your email's trash can at first. If you can't find verify email in trash can, please try send verify email again.
 
-如果仍然收不到验证邮件，请检查你的账号绑定的邮箱是否正确，如果不正确则将其更改为正确的邮箱。
+If you still can't receive verify email, please check your email that bind your account is correct, if incorrect, please change it to your correct email.
 
-如果你确定你绑定的邮箱正确，并且验证邮件不在你的邮箱的垃圾箱里，请加入我们的 [官方用户交流群](/user-group.html)，联系站点管理员处理。
+If you sure your email that bind your account is correct, and verify email can't find in your trash can, please join our [User Group](/user-group.html), contact staff to process.
 
-### 我的角色被别人占用了，我可以要回来吗？
+### My profile are have by others, I can get it back?
 
-不可以。重名的几率很大，随意更改角色所有者对双方都不公平，所以不管你是名声多大的主播还是某个达官贵人家的孩子，我们都不会提供更改角色所有者的服务。
+No. The chance of having the same name is very high, change players' owner is unfair to both users, so whether you are an anchor or a rich people, we don't provide the service that change players' owner.
 
-如果你发现你的角色名被别人占用了，你可以先联系当前该角色的所有者进行协商，没准这事儿就成了，同时你还收获了一个朋友呢。如果你好好地和当前该角色的所有者商量（而不是一上来就用一种强硬的口气要求释放角色——这种情况大多不会有好结果），在大多数情况下，他们都可以是很好说话的。至于怎么联系？缘分到了就自然联系上了咯。
+If you find that your player's name has been occupied by others, you can contact this player's owner. If you talk to this player's owner properly (not use a strong tone to talk to this player's owner) , in many times, they are friendly. But how to contact? By fate.
 
-有一个例外情况是你拥有正版 Minecraft 并且你的正版 ID 就是被他人占用的角色名。这种情况下，你可以通过 [绑定正版角色](/newbee/player.html#绑定正版角色) 来强制获取这个角色的所有权。
+There is a special case: you have Genuine Minecraft account, and your Genuine Minecraft ID just the player's name that be occupied by others. In this condition, you can pass [Bind your Genuine Minecraft Account](/newbee/player.html) to get this player's owner.
```

Comment:
However, there is a special case: you have a Mojang account with Genuine Minecraft, and your Genuine Minecraft player name is just the player name that be taken by others. In this situation, you can get your profile by [Bind Your Mojang Account](/newbee/player.html).